applications/zigbee_weather_station/README.rst

```diff
@@ -0,0 +1,512 @@
+You can use one of the following options for setting up the Zigbee network coordinator:
+
+* For `Testing`_ purposes, program the :ref:`Zigbee shell <zigbee_shell_sample>` as the network coordinator to be able to use the interactive shell for readings and bindings.
+  Read the sample documentation for more information.
+* For generic usage of the application, you can use the standard :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>`.
+
+To program a Thingy:53 device where the preprogrammed MCUboot bootloader has been erased, you need the external J-Link programmer.
+If you own an nRF5340 DK that has an onboard J-Link programmer, you can also use it for this purpose.
+
+If the Thingy:53 device is programmed with a Thingy:53-compatible sample or application, then you can also update the firmware using MCUboot's serial recovery or DFU over Bluetooth LE.
+See :ref:`thingy53_app_guide` for details.
+
+For capturing packets while testing, you can use `nRF Sniffer for 802.15.4`_ with `Wireshark configured for Zigbee <Configuring Wireshark for Zigbee>`_.
+
+User interface
+**************
+
+LED (LD1):
+    Shows the overall state of the device and its connectivity.
+    The following states are possible:
+
+    * ``release`` build type
+
+      * Even flashing (red color, 500ms on/500ms off) - The device is in the Identify mode (after network comissioning).
+
+    * ``debug`` build type
+
+      * Constant light (blue) - The device is connected to a Zigbee network.
+      * Even flashing (red color, 500ms on/500ms off) - The device is in the Identify mode (after network comissioning).
+
+    .. note::
+       Thingy:53 allows you to control RGB components of its single LED independently.
+       This means that the listed color components can overlap, creating additional color effects.
+       For example, **LED (LD1)** can indicate that the device is connected to a Zigbee network in blue, but if the device goes into the Identify mode at the same time, the flashing to indicate this will be purple, not red.
+
+Power switch (SW1):
+    Used for switching Thingy:53 on and off.
+
+Button (SW3):
+    Used to start and cancel the Identify mode.
+    Additionally, pressing the button indicates user input, which allows a Sleepy End Device to start the network rejoin procedure if it was stopped (for example, after timeout).
+
+USB port:
+    Used for getting logs from the device.
+    It is enabled only for the debug configuration of the application.
+    See the `Selecting a build type`_ section to learn how to select the debug configuration.
+
+Configuration
+*************
+
+|config|
+
+The following options in :file:`Kconfig` can be adjusted:
+
+* ``FIRST_WEATHER_CHECK_DELAY_SECONDS`` - delay from application initialization to first sensor data readout, in seconds; default value: ``5``
+
+* ``WEATHER_CHECK_PERIOD_SECONDS`` - period of cyclic sensor readouts after the first readout, in seconds; default value: ``60``
```

Comment:
```suggestion
.. _zigbee_weather_station_app_configuration_readout:

Readout configuration options
=============================

The following Kconfig option values can be set to custom values:

* :kconfig:`CONFIG_FIRST_WEATHER_CHECK_DELAY_SECONDS` - This option defines the delay from the moment of the application initialization to the first sensor data readout.
  The value is provided in seconds, with the default value set to ``5``.

* :kconfig:`WEATHER_CHECK_PERIOD_SECONDS` - This option defines the period of cyclic sensor readouts after the first readout.
  The value is provided in seconds, with the default value set to ``60``.
```
This needs to be a separate section, as for other applications or samples.

Reply:
For some reason I cannot apply this change in github gui like for other changes... I will try adding it manually

Reply:
Done manually.